### PR TITLE
Move overall diagram to the context section

### PR DESCRIPTION
**Pull request by jakobw**
This moves the overview diagram out of the building block view and into the business context section. It also shows it on the docs' homepage as the entry point diagram. The only change I made to the diagram was to change the label of the quickstatements user arrow from "tba interaction" to "submits edits".

I stumbled a little bit over the direction of the arrows in this diagram. The other context diagrams show data flow, whereas this one shows control flow.

Bug: [T274240](https://phabricator.wikimedia.org/T274240)

### CODE REVIEW

File: systems/overview/diagrams/03-business-context.drawio.svg [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    <defs/>
+    <g>
+        <ellipse cx="584" cy="23.5" rx="7.5" ry="7.5" fill="#dae8fc" stroke="#6c8ebf" pointer-events="all"/>
+        <path d="M 584 31 L 584 56 M 584 36 L 569 36 M 584 36 L 599 36 M 584 56 L 569 76 M 584 56 L 599 76" fill="none" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="all"/>
+        <g fill="#000000" font-family="Helvetica" text-anchor="middle" font-size="12px">
+            <text x="583.5" y="10.5">
+                Client Editor / Viewer
```
Comment (addshore) [inline systems/overview/diagrams/03-business-context.drawio.svg:8]:
```suggestion
                Client Editor
```

File: systems/overview/diagrams/03-business-context.drawio.svg [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        <g fill="#ffffff" font-family="Helvetica" text-anchor="middle" font-size="12px">
+            <text x="583.5" y="170.5">
+                Wikibase Client
+            </text>
+        </g>
+        <ellipse cx="399" cy="23.5" rx="7.5" ry="7.5" fill="#dae8fc" stroke="#6c8ebf" pointer-events="all"/>
+        <path d="M 399 31 L 399 56 M 399 36 L 384 36 M 399 36 L 414 36 M 399 56 L 384 76 M 399 56 L 414 76" fill="none" stroke="#6c8ebf" stroke-miterlimit="10" pointer-events="all"/>
+        <g fill="#000000" font-family="Helvetica" text-anchor="middle" font-size="12px">
+            <text x="398.5" y="10.5">
+                Wikidata/Wikibase Editor/Viewer
```
Comment (addshore) [inline systems/overview/diagrams/03-business-context.drawio.svg:163]:
```suggestion
                Wikidata/Wikibase Editor
```

**Review by addshore — CHANGES_REQUESTED** [review]

**Comment by addshore** [comment]
2 changes requested per the fact I just merged https://github.com/wmde/wikidata-wikibase-architecture/pull/93 which removed the "Viewer" words everywhere

File: systems/overview/diagrams/03-business-context.drawio.svg (hunk 1 above)
Comment (jakobw) [reply systems/overview/diagrams/03-business-context.drawio.svg]: Whoopsie, thanks for spotting that. Fixed!

**Review by addshore — APPROVED** [review]

**Review by Silvan-WMDE — APPROVED** [review]

**Pull request merged**